spec/controllers/organizations_controller_spec.rb

```diff
@@ -19,6 +20,50 @@
       end
     end
 
+    describe "as owner" do
+      before {login(user)}
+      it "display unpublished organizations" do
+        user.organization_id = unpublished_organization.id
+        user.save
+        get :show, id: unpublished_organization.id
+        expect(assigns(:organization)).to eq(unpublished_organization)
+      end
+      it "display published organizations" do
+        user.organization_id = published_organization.id
+        user.save
+        get :show, id: published_organization.id
+        expect(assigns(:organization)).to eq(published_organization)
+      end
+    end
+
+    describe "as user that owns another organization" do
+      before do
+        unpublished_organization
+        login(user)
+      end
+      it "doesn't display unpublished organizations" do
+        expect {get :show, id: unpublished_organization.id }.to raise_exception(ActiveRecord::RecordNotFound)
```

Comment:
Add a space between `{` and `get` -> `{ get...`, check for any other similar inconsistencies in your tests for this PR